.gitignore

```diff
@@ -52,6 +52,7 @@ wheels/
 .installed.cfg
 *.egg
 src/lightning/*/
+src/*/version.info
```

Comment:
Why do we gitignore this? We want to track it

Reply:
no, we want to track only the global in `src/version.info` otherwise this PR does not have any value :)